src/tail_csv.c

```diff
@@ -136,60 +142,24 @@ static _Bool tcsv_check_index (int index, size_t fields_num, char const *name)
 static int tcsv_read_buffer (instance_definition_t *id,
         char *buffer, size_t buffer_size)
 {
-    char **metrics;
     size_t metrics_num;
 
+    char *saveptr;
```

Comment:
It's better to init saveptr to NULL, on some systems compiler might complain about use of uninitialized pointer. This is just issue I had once on collectd CI.